CDP4Dal/ISession.cs

```diff
@@ -186,18 +186,47 @@ public interface ISession
         Task Open();
 
         /// <summary>
-        /// Read an <see cref="Iteration"/> from the underlying <see cref="IDal"/> and set the active <see cref="DomainOfExpertise"/> for the <see cref="Iteration"/>.
+        /// Open the underlying <see cref="IDal"/> and update the Cache with the retrieved objects.
+        /// </summary>
+        /// <returns>
+        /// an await-able <see cref="Task"/>
+        /// </returns>
+        /// <remarks>
+        /// The <see cref="CDPMessageBus"/> is not used to send messages to notify listeners of <see cref="Thing"/>s that
+        /// have been added to the cache.
+        /// </remarks>
+        Task OpenSilently();

```

Comment:
my preference would be to not have an extra method on this interface and implementation. Please use

`Task Open(bool activeMessageBus = true);`

Then we do not need an extra method that in my opinion will cause confusion.